components/about/TeamMember.vue

```diff
@@ -123,6 +123,7 @@ export default {
 
 .member-image > * {
   width: 16rem;
+  border-radius: 50%;
```

Comment:
The images have been grouped now on Figma and you should be able to export them with the circular masking. 

This works too, so no issues if you'd like to keep the current changes.